compiler/assembler.h

```diff
@@ -0,0 +1,25 @@
+// vim: set ts=8 sts=2 sw=2 tw=99 et:
+//
+//  Pawn compiler - Error message system
```

Comment:
Every new header file has this comment about an `Error message system`. Might want to remove that / change it.

Reply:
Thanks, I'll fix this up in the next commit.